src/common/plugin.ts

```diff
@@ -84,12 +85,18 @@ export abstract class Plugin implements IPlugin {
     const totalChannels = channels.length;
     channels.splice(this._numChannelsShown);
 
+    if (message instanceof CommandInteraction) {
+      this.container.guildService.getChannel('test')
```

Comment:
?

Reply:
I'm removing this in my branch; I think it was just for testing